lib/SILOptimizer/IPO/CrossModuleOptimization.cpp

```diff
@@ -434,13 +457,18 @@ bool CrossModuleOptimization::canSerializeFunction(
     return false;
 
   // Check if any instruction prevents serializing the function.
+  InstructionVisitor visitor(*function, *this, true);
   for (SILBasicBlock &block : *function) {
     for (SILInstruction &inst : block) {
+      visitor.getBuilder().setInsertionPoint(&inst);
+      visitor.visit(&inst);
```

Comment:
How is the result of this visit considered?

Reply:
see the updates